Pipfile

```diff
@@ -59,6 +59,7 @@ psycopg2-binary = "*"
 sphinx = "*"
 sphinx-rtd-theme = "*"
 python-dotenv = "*"
+faker = "*"
```

Comment:
We should probably also move this to the dev dependencies. Not something we want to do in Production.